plan_runner/manipulation_station_simulator.py

```diff
@@ -59,12 +54,15 @@ def RunSimulation(self, plans_list, gripper_setpoint_list,
                       extra_time=0, real_time_rate=1.0, q0_kuka=np.zeros(7), is_visualizing=True):
         '''
         Constructs a Diagram that sends commands to ManipulationStation.
-        :param plans_list:
-        :param gripper_setpoint_list:
-        :param extra_time:
-        :param real_time_rate:
-        :param q0_kuka:
-        :return:
+        :param plans_list: A list of Plans to be executed.
```

Comment:
It seems like the preferred way to list params is with @param

https://github.com/RobotLocomotion/drake/blob/ec7c62d3e44b79d8e54d75b9c88481c1e8905c4e/examples/manipulation_station/end_effector_teleop.py#L128